setup/depends.sh

```diff
@@ -17,69 +17,64 @@
 # #########################
 #     value
 # #########################
-pgram=brew
```

Comment:
This variable was set and used in two places without change and, in one instance, was misspelled.